.ddev/config.yaml

```diff
@@ -1,5 +1,6 @@
 APIVersion: v1.8.0
 name: introduction
+project_tld: ddev.local
```

Comment:
The database dump contained the old `.ddev.local` string as well so i decided to adapt that one line in configuration instead of changing and having to test many in the code and in the database. But if the `.ddev.site` is preferred for some reason, then i would do that refactoring and testing, yes.

Reply:
`.ddev.local` refactored to `.ddev.site`